src/controllers/controller.bubble.js

```diff
@@ -154,7 +155,8 @@ module.exports = function(Chart) {
 				'hoverBorderWidth',
 				'hoverRadius',
 				'hitRadius',
-				'pointStyle'
+				'pointStyle',
+				'rotation'
```

Comment:
I think this would also need to be `pointRotation` if @simonbrunel's comment above is addressed